aidants_connect_web/templates/aidants_connect_web/id_provider/authorize.html

```diff
@@ -7,6 +7,7 @@
 {% block extracss %}
   <link href="{% static 'css/id_provider.css' %}" rel="stylesheet" />
   <link href="{% static 'css/users_search.css' %}" rel="stylesheet" />
+  <link href="{% static 'css/jquery-ui.css' %}" rel="stylesheet" />
```

Comment:
```suggestion
  <link href="https://code.jquery.com/ui/1.13.2/themes/base/jquery-ui.css" rel="stylesheet" />
```

Et faudra rajouter `"https://code.jquery.com/ui/1.13.2/themes/base/jquery-ui.css"`  au tuple `CSP_STYLE_SRC` dans `settings.py` et supprimer le fichier `aidants_connect_web/static/css/jquery-ui.css`.